### ISSUE TITLE
Elevated factor VIII level

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7404]
**Preferred term label:**
Elevated factor VIII level

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 301071 

https://pubmed.ncbi.nlm.nih.gov/33275657/  7/7

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
This was Increased factor VIII activity HP:0030977, adding synonym
